mautic/mautic
# Manual lead merging

**Description**
This adds a option on the lead viewing page to be able to merge the current lead with another selected one.
It is simply a interface for the already built back-end functionality, one adverse feature is that the merged lead gets a new ID, campaigns are kept though.

Fixes the issue #792 I created, I have finally have my head around Symfony

**Testing**

1. View a lead
2. Select the "Merge" option from under the "Edit" dropdown
3. Select/search for the lead you wish to merge with this one
4. The earliest added leads field data should remain and the others be merged into this
